src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -50,41 +47,36 @@ public void addAll(List<T> list) {
     @Override
     public T get(int index) {
         checkIndex(index);
-        return node(index).item;
+        return nodeByIndex(index).item;
     }
 
     @Override
     public T set(T value, int index) {
         checkIndex(index);
-        Node<T> x = node(index);
-        T oldVal = x.item;
-        x.item = value;
+        Node<T> node = nodeByIndex(index);
+        T oldVal = node.item;
+        node.item = value;
         return oldVal;
     }
 
     @Override
     public T remove(int index) {
         checkIndex(index);
-        T element = node(index).item;
-        unlink(node(index));
+        T element = nodeByIndex(index).item;
```

Comment:
create local variable so you will not need to call method nodeByIndex(index) all the time